modal/_utils/git_utils.py

```diff
@@ -0,0 +1,75 @@
+# Copyright Modal Labs 2025
+import os
+import subprocess
+from typing import Dict, List, Optional, Union
+
+
+def run_command(args: List[str]) -> Optional[str]:
+    """Run a command and return its output.
```

Comment:
Naming nit, but maybe something like `run_command_failible` to capture the "return None on any error" semantics.